src/Filter/EmptyFilter.php

```diff
@@ -15,14 +15,16 @@
 
 /**
  * @psalm-suppress InvalidExtendClass
+ *
+ * @deprecated since sonata-project/doctrine-orm-admin-bundle version 3.27 and will be removed in 4.0.
  */
 final class EmptyFilter extends NullFilter
 {
     public function __construct()
     {
         // NEXT_MAJOR: remove this file
         @trigger_error(sprintf(
-            'The %s class is deprecated since version 3.27 and will be removed in 4.0.'
+            'The %s class is deprecated since sonata-project/doctrine-orm-admin-bundle version 3.27 and will be removed in 4.0.'
```

Comment:
```suggestion
            'The "%s" class is deprecated since sonata-project/doctrine-orm-admin-bundle version 3.27 and will be removed in 4.0.'
```